firmwares/Config.ini

```diff
@@ -0,0 +1,5 @@
+NameWifiNINABFWifiNINA firmware Version1.0.0BoardsArduino MKR WiFi 1010;Arduino MKR Vidor 4000;Arduino UNO WiFi REV2:
```

Comment:
cfg file is fine but change file format please

Reply:
Maybe JSON?